agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/BeforeAgentInstaller.java

```diff
@@ -187,6 +195,54 @@ public void run() {
                 }
             }
         });
+
+        pollJsonConfigEveryMinute();
+    }
+
+    private static volatile Long lastModifiedTime;
+    private static void pollJsonConfigEveryMinute() {
+        Thread pollingThread = new Thread(new JsonConfigPolling());
+        pollingThread.setDaemon(true);
+        Executors.newSingleThreadScheduledExecutor().scheduleWithFixedDelay(pollingThread, 60, 60, SECONDS);
```

Comment:
this is a bit more consistent with other places in code:

Executors.newSingleThreadScheduledExecutor(ThreadPoolUtils.createDaemonThreadFactory(JsonConfigPolling.class))
.scheduleWithFixedDelay(...)

and then don't need to create `Thread`

Reply:
scheduleWithFixedDelay requires a runnable.  I also looked into the implementation of ThreadPoolUtils.. it created a new thread and set daemon to true.  it's essentially the same..

Reply:
it's mostly for threadFactory and threadPool.

Reply:
`ThreadPoolUtils` creates a `ThreadFactory` (not a `Thread`), which is a more typical way to configure a thread pool

Reply:
yup, spotBug complained too: DMI_THREAD_PASSED_WHERE_RUNNABLE_EXPECTED.  😢

Reply:
Updated.